### DISCUSSION TITLE
How I do keep my azure blob storage key ‘MINIO_SECRET_KEY’ safe

### DISCUSSION THREAD

**Discussion by ep23-eps** [minio/minio#19577] [Q&A]
I used helm chat to configure Minio agent to access my azure blob container, it’s for backup log access. in environmental variable I used blob container name and access key.I don’t want share my secret key as password to users to login Minio. Help me to how do I restrict that?

Br,
Nagavardhan.G

**Comment by klauspost** [comment]
Set up a user/service account: https://min.io/docs/minio/kubernetes/upstream/administration/identity-access-management/minio-user-management.html#id1
